addons/l10n_bd/models/template_bd.py

```diff
@@ -0,0 +1,47 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+from odoo.addons.account.models.chart_template import template
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = 'account.chart.template'
+
+    @template('bd')
+    def _get_bd_template_data(self):
+        return {
+            'property_account_receivable_id': 'l10n_bd_100201',
+            'property_account_payable_id': 'l10n_bd_200101',
+            'property_account_expense_categ_id': 'l10n_bd_500200',
+            'property_account_income_categ_id': 'l10n_bd_400100'
+        }
+
+    @template('bd', 'res.company')
+    def _get_bd_res_company(self):
+        return {
+            self.env.company.id: {
+                'account_fiscal_country_id': 'base.bd',
+                'bank_account_code_prefix': '100105',
+                'cash_account_code_prefix': '100108',
+                'account_default_pos_receivable_account_id': 'l10n_bd_100202',
+                'account_journal_suspense_account_id': 'l10n_bd_100102',
+                'account_journal_payment_debit_account_id': 'l10n_bd_100103',
+                'account_journal_payment_credit_account_id': 'l10n_bd_100104',
+                'transfer_account_id': 'l10n_bd_100101',
+                'account_journal_early_pay_discount_loss_account_id': 'l10n_bd_501107',
+                'account_journal_early_pay_discount_gain_account_id': 'l10n_bd_400304',
+                'account_sale_tax_id': 'VAT_S_IN_BD_10',
+                'account_purchase_tax_id': 'VAT_P_IN_BD_10'
+            },
+        }
+
+    @template('bd', 'account.journal')
+    def _get_bd_account_journal(self):
+        return {
+            "tax_adjustment": {
+                "name": "Tax Adjustments",
+                "code": "TA",
+                "type": "general",
+                "show_on_dashboard": True,
+                "sequence": 1,
```

Comment:
I think we shouldn't put sequence 1, if you create a misc entry, it will default to this one, while I don't think that's what we would want.

Reply:
Yes, that's not what we want.
But small question here, How we choose the sequence like Do we have a roll of something like that or even an interval?
I do understand why we should have it as 1, but I'm a bit afraid like changing it to any number else could have the same issue with other journal.

Reply:
So, the default defined in `account_journal.py` is 10 (if you don't define one).
You can have the same sequence as another. Then, to separate the journals, it takes the `_order` defined on the model (still in `account_journal.py`), here sequence, then type, then code.
Here, I would think that the default (10) is good enough.
The rest of the journal sequences are defined here https://github.com/odoo/odoo/blob/17.0/addons/account/models/chart_template.py#L945

Reply:
Thanks <3